verl/utils/import_utils.py

```diff
@@ -57,6 +57,16 @@ def is_nvtx_available():
         nvtx_spec = importlib.util.find_spec("nvtx")
     except ModuleNotFoundError:
         nvtx_spec = None
+
+    from verl.utils.device import is_cuda_available
+
+    if is_cuda_available and nvtx_spec is None:
+        warnings.warn(
+            "WARNING: nvtx is not available in CUDA platform. Please 'pip3 install nvtx'",
```

Comment:
would it generate unrelated warning even if profiler is not turned on?

Reply:
We found users forget to install nvtx and the profiler not works well. I make it an assertion rather than a warning now, if users intend to do profile.